manifest.yml

```diff
@@ -3,6 +3,4 @@ applications:
 - name: carbon-storybook
   memory: 64M
   buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
-  routes:
-  - route: carbon-react-storybook.mybluemix.net
-  - route: react.carbondesignsystem.com
+  random-route: true
```

Comment:
Could you revert this one back after done testing with the staging URL? 🙂

Reply:
Yep! good catch 👍